FormattingStyleGuide.md

```diff
@@ -0,0 +1,43 @@
+# Formatting & Style Guide
+
+## Scope and purpose
+This document provides formatting and style conventions for publishing documents in __GitHub__ in __Markdown__ format. Some advice on document conversion and how to meet more sophisticated publishing needs is offered.
+
+## General
+- Use [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).
+- If the workflow prescribes the document to go into BitBucket repository first, then you may find [BitBucket Markdown Syntax Guide](https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html) useful.
+
+## Title & headings
+- Use # (h1) for document title.
+- Use ## (h2) for top-level headings.
+- Number the headings manually. There's no heading numbering support in Markdown.
```

Comment:
Referencing can be done using markdown links. I did this for the system parameters document and the references to headings works as hyperlinks. The document can be previewed here: https://github.com/vrk-kpa/xroad/blob/markdeep-test/xtee6/doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

Are there other reasons to remain numbering?

I also tried http://casual-effects.com/markdeep format and it looks good on a browser, but unfortunately github does not render it. The document syntax is almost like markdown with minor changes and a line of javaScript at the bottom of the document. The sample is here: https://github.com/vrk-kpa/xroad/blob/markdeep-test/xtee6/doc/Manuals/ug-syspar_x-road_v6_system_parameters.md.html and because github does not render it, you need to download it as file and open the file in a browser to see the document rendered correctly.